src/tools/recvfuzz/recvfuzz.cpp

```diff
@@ -127,14 +209,14 @@ UdpUnreachCallback(
 struct TlsContext
 {
     CXPLAT_TLS* Ptr {nullptr};
-    CXPLAT_SEC_CONFIG* SecConfig {nullptr};
+    CXPLAT_SEC_CONFIG* ClientSecConfig {nullptr};
+    CXPLAT_SEC_CONFIG* ServerSecConfig {nullptr};
     CXPLAT_TLS_PROCESS_STATE State;
     uint8_t AlpnListBuffer[256];
 
-    TlsContext()  {
+    void CreateContext(uint64_t initSrcCid = MagicCid)  {
```

Comment:
Why a function and not a constructor?

Reply:
There was a problem with scope, the constructor was initialising the TLS context when I didnt want it to.